plugins/extend-give-wp/src/views/donation-form-1349/featured-image-view.php

```diff
@@ -0,0 +1,11 @@
+<?php
+/*
+ * The donation form featured image view file.
+ */
+namespace spiralWebDB\ExtendGiveWP;
+
+?>
+<figure id="attachment_<?php echo esc_attr( $postID ) ?>" class="donation-form-featured-image attachment_<?php echo esc_attr( $postID ) ?>" aria-describedby="caption-attachment-<?php echo esc_attr( $postID ) ?>">
+	<?php echo wp_get_attachment_image( 1411, $size = 'large', $icon = false, $attr = array( 'class' => 'featured-image attachment_1411 ID-1411' ) ); ?>
```

Comment:
Don't hardcode the attribute ID. Instead look it up in the source code that loads this view file.

Reply:
@hellofromtonya On a separate but related matter, the custom callback could pass `$attachment_id` as an argument, and the value of that parameter could be passed from a custom configuration. 

The view called within the callback passes `$attachment_id` as an argument to the WordPress API function `wp_get_attachment_image()` ( located at `/wp-includes/media.php` on L#983 ). That would allow a refactor within the callback where we could get rid of lines 25 & 26 above. 

The `$attr` argument of `wp_get_attachment_image()` allows one to pass custom classes to the `<img>` tag. Since we want to stop using specific references to post or attachment IDs, the classes `attachment_1411 ID-1411` should be removed from the view on L#9.

Reply:
@hellofromtonya See commit  [8dc43c7](https://github.com/rgadon107/cornerstone/pull/52/commits/8dc43c774c31a25b8f05f38d1881abf17003e06e) for an update on this issue. 

- References to a specific post ID is replaced with the argument.
- The value of the `$size` argument was passed to the custom callback.
- The values of the `$attr` argument that reference a specific attachment or post ID was deleted.